src/Ganeti/HTools/CLI.hs

```diff
@@ -762,6 +765,14 @@ oSkipNonRedundant =
     "Skip nodes that host a non-redundant instance",
     OptComplNone)
 
+oStaticKvmNodeMemory :: OptType
+oStaticKvmNodeMemory =
+  (Option "" ["static-kvm-node-memory"]
+   (reqWithConversion (tryRead "static node memory")
+    (\i opts -> Ok opts { optStaticKvmNodeMemory = i }) "N")
+   "use static node memory [in MB] on KVM instead of value reported by hypervisor.",
```

Comment:
The value overrides hypervisor reported values if it is greater than 0. Documentation was extended.